tyk/portal.go

```diff
@@ -14,6 +14,11 @@ import (
 
 // BootstrapClassicPortal bootstraps Tyk Classic Portal.
 func (s *Service) BootstrapClassicPortal() error {
+	if s.appArgs.Tyk.Admin.Auth == "" {
+		s.l.Warn("Missing Admin Auth configuration may cause Authorization failures on Tyk",
+			"If Tyk Dashboard bootstrapping is disabled, please provide Admin Auth through environment variables")
+	}
```

Comment:
Shouldn't it return after the warning?

Reply:
yes, we can. eventually, the subsequent http calls will fail due to missing authorization field.

Reply:
addressed in bafdb015cfd13b9934b6802058bc838d2eb9bc14